pyfolio/tears.py

```diff
@@ -328,16 +330,20 @@ def create_position_tear_sheet(returns, positions, gross_lev=None,
         If True, returns the figure that was plotted on.
     set_context : boolean, optional
         If True, set default plotting style context.
+    sector_mapping:
+        Dictionary or Series.
```

Comment:
The type info should be moved up (see the others) and renamed to "dict or pandas.Series". Then the second line describes the data structure expected and what it results in.